src/utils/mesh.js

```diff
@@ -241,6 +234,74 @@ export async function createMesh (runAIOCommand, installedPlugins) {
   }
 }
 
+/**
+ *
+ * @param runAIOCommand
+ * @param installedPlugins
+ * @param meshUrl
+ */
+export async function updateMesh (runAIOCommand, installedPlugins, meshUrl) {
+  try {
+    await checkAndInstallMeshPlugin(installedPlugins)
+
+    const { apiKey, datasource } = config.get('commerce')
+    const { saas, paas, catalog } = datasource
+
+    console.log('Updating API Mesh...')
+
+    await createTempMeshConfigFile(saas, paas, catalog, apiKey)
+
+    await runAIOCommand('api-mesh:update', [meshConfigFilePath, '-c'])
```

Comment:
Let's add a confirmation for this, and maybe some log about where the mesh points to before this.
It's a destructive operation, so want users to be sure about what they are doing.

For example:

```
> Mesh found for {project}, {workspace}, {org} are you sure you want to overwrite? y/n
> Updating mesh. Previous source: {source}. New source: {source}

Reply:
Got it, mesh already does that by default. I have removed the `-c` flag to avoid auto confirm and let the user confirm the update action.

Reply:
Nah, there isnt an option for rollback. They will have to update with an older config. Most of the mesh customers use git for version control so they will have access to an older mesh config file which they can use to update.

We are working on a rollback / history command, meanwhile this would be their option.